### PR TITLE
improved airflow connections population

### PR DESCRIPTION
**Pull request by comtihon**
Closes https://github.com/comtihon/catcher_modules/issues/64

Fixed aws population
Fixed extras population for db-based connections
Made connection be populated only if it has `type`

### CODE REVIEW

File: catcher_modules/utils/db_utils.py [hunk 1]
```diff
@@ -12,21 +12,52 @@
...
     from sqlalchemy import create_engine
     return create_engine(conf_str)
 
 
+def __construct_str_configuration(conf, dialect):
+    return '{}://{}:{}@{}:{}/{}'.format(dialect,
+                                        conf['user'],
+                                        conf['password'],
+                                        conf['host'],
+                                        conf.get('port', default_ports.get(dialect.lower(), '')),
```
Comment (kbelova) [inline catcher_modules/utils/db_utils.py:57]: semicolon should be added together with port only in cases when port is provided from configuration

**Review by kbelova — CHANGES_REQUESTED** [review]
Minor issues to fix. Rest looks good. Thank you for your time!

File: catcher_modules/database/mongo.py [hunk 2]
```diff
@@ -169,15 +190,23 @@ def action(self, includes: dict, variables: dict) -> any:
...
+            client = MongoClient(conf['url'])
+            database = client.get_database('test')
         else:
             database = conf.pop('database', 'test')
+            conf.pop('type', None)  # removes airflow step specific field
+            conf.pop('extra', None)  # removes airflow step specific field
             client = MongoClient(**conf)
             database = client.get_database(database)
             if database is None and conf['database'] is not None:
                 database = client.get_database(conf['database'])
```
[the remark this answers is not in the document]
Comment (comtihon) [reply catcher_modules/database/mongo.py]: thanks!

File: catcher_modules/utils/db_utils.py (hunk 1 above)
Comment (comtihon) [reply catcher_modules/utils/db_utils.py]: resolved

**Review by kbelova — APPROVED** [review]
LGTM, @comtihon  lets merge it

**Pull request merged**